### ISSUE TITLE
ClimateTriage also lists closed issues for projects

### ISSUE THREAD

**Issue by killerog** [protontypes/climate-triage#46]
Hi,

Most of the issues listed on the site are actually open, but sometimes I come across one that's already closed: https://github.com/klimatbyran/klimatkollen/issues/508

I don't know if this is really a bug or just an issue that slipped through an automated process. I assume these don't get updated manually?

**Comment by andrew** [comment]
@killerog I should probably be syncing climatetriage issues more frequently, sorry about that, I'll add it to my todo list

**Comment by killerog** [comment]
No worries, I don't know how much manual labour it requires.

I did try to run the project locally to see if I could help, and I couldn't find the project anymore under the language I found it under originally. It seems to have moved from HTML to TypeScript. Maybe that's something you can update at the same time, to save you some work?

**Comment by Codeshark-NET** [comment]
Hey there!

I just noticed the github-action we had in place, for actually updating the data via ecosyste.ms, was disabled and hadn't run in quite some time - which would explain the outdated data. 

I just re-enabled and deployed latest data, which looks updated to me now. Normally, it would update every hour.

Could you confirm @killerog? Good catch 🙏

**Comment by killerog** [comment]
I still see some other closed issues showing up: https://github.com/klimatbyran/klimatkollen/issues/324 

The categories and number of issues have been updated though, so maybe I'm looking at a cache?

**Comment by Codeshark-NET** [comment]
Yes, very likely. Sorry for the delay. Haven't been able to reproduce since then, so closing this issue for now, thanks again! 👍
